yield_plugin/src/yield_plugin.cpp

```diff
@@ -450,15 +466,21 @@ namespace yield_plugin
                                                                                                 goal_accel,
                                                                                                 initial_time,
                                                                                                 planning_time);
-    RCLCPP_DEBUG_STREAM(nh_->get_logger(),"Used original_max_speed: " << original_max_speed);
+    RCLCPP_DEBUG_STREAM(rclcpp::get_logger("yield_plugin"),"Used original_max_speed: " << original_max_speed);
+    for (size_t i = 0; i < polynomial_coefficients.size(); i++) {
+      RCLCPP_DEBUG_STREAM(rclcpp::get_logger("yield_plugin"),"Coefficient " << i << ": " << polynomial_coefficients[i]);
+    }
     const auto smallest_time_step = get_smallest_time_step_of_traj(original_tp);
+    int new_traj_idx = 1;
+    int original_traj_idx = 1;
+    bool is_accelerating = (initial_velocity < goal_velocity) ? true : false;
```

Comment:
I dont see where is_accelerating is used?

Reply:
Oh good catch this was an artifact from a different method I was trying